tests/TestCase/Collection/CollectionTest.php

```diff
@@ -1769,6 +1769,35 @@ public function testSumOfCallable($items)
         $this->assertEquals(600, $sum);
     }
 
+    /**
+     * Provider for sumOf float tests
+     *
+     * @return array
+     */
+    public function sumOfFloatProvider()
```

Comment:
If there's an existing provider method for `sumOf()` tests it would be better to use the same for float values to.

Reply:
Still getting used to php unit test providers.

Reply:
No worries :)